SCClassLibrary/Common/GUI/Base/QWebView.sc

```diff
@@ -253,6 +253,27 @@ WebView : View {
 	contentsSize { ^this.getProperty('contentsSize') }
 
 	scrollPosition { ^this.getProperty('scrollPosition') }
+	scrollPosition_ { |point| ^this.setProperty('scrollPosition', (point / this.zoom)) }
+
+	scrollUp { |value|
+		var position = this.scrollPosition;
+		this.scrollPosition_(
+			Point(
+				position.x,
+				(position.y - value).max(0)
+			)
+		);
+	}
+
+	scrollDown { |value|
+		var position = this.scrollPosition;
+		this.scrollPosition_(
+			Point(
+				position.x,
+				(position.y + value).min(this.contentsSize.height - this.bounds.height)
+			)
+		);
+	}
```

Comment:
scrollUp and scrollDown are almost the same function... how about combining them? for instance:
```suggestion
	scrollUp { |value|
		this.scrollDown(value.neg)
	}

	scrollDown { |value|
		var position = this.scrollPosition;
		this.scrollPosition_(
			Point(
				position.x,
				(position.y + value).clip(0, this.contentsSize.height - this.bounds.height)
			)
		);
	}
```

Reply:
ok. thank you, good suggestion. It is rewrited and I also added new key shortcut F5 to reload page. Useful when workong on documentation. Tested on Linux.